src/content/blog/ga_qas.md

```diff
@@ -0,0 +1,224 @@
+| crossover_func | crossover.onepoint_crossover | $f$: qiskit.QuantumCircuit $\times$ qiskit.QuantumCircuit $\rightarrow$ qiskit.QuantumCircuit $\times$ qiskit.QuantumCircuit |
+| mutate_func | mutate.layerflip_mutate | $f$: qiskit.QuantumCircuit $\rightarrow$ qiskit.QuantumCircuit |
+| threshold_func | threshold.compilation_threshold | $f$: $\mathbb{R} \rightarrow$ {0,1} |
+
+_**Table 2**: Parameter functions for GA-QAS._
+
+# Step 3. Run GA-QAS
+
+You can call the method `evol()` to start running GA-QAS. 
+```
+env.evol(
+   verbose = 1, 
+   auto_save = True
+)
+```
+There are only two parameters: the first is verbose (1 means print % process, 0 means no print), and the second is the saving option.
+The saving result's folder is detailed on [GA-QAS: folder result](https://github.com/vutuanhai237/qoop/wiki/GA%E2%80%90QAS:-folder-result).
+# Step 4. Plot result
+
+Currently, we support plotting fitness values against a number of generations. In the future, we will develop more presentation plots.
+
+```
+env.plot()
+```
+
+<img alt="" src="/public/images/ga-qas/ga-qas3.png">
+
+_Fig 3. Fitness values versus number of generations._
+
+The result is saved in a folder; the default folder name is based on the fitness function name. We care about a file named `best_circuit.qpy`, which is our final solution. Then, we can load it by $\langle qo|op\rangle$ and put it into fitness again to test:
+
+```
+from qoop.backend import utilities
+best_circuit = utilities.load_circuit("Path to best_circuit.qpy")
+fitness_value = fitnessW(best_circuit)
+```
+
+# Conclusion
+
+In this post, we have introduced a way to use GA-QAS, just define your own problem (fitness function) and it will help you find best ansatz automatically. For more information and documentation, explore the following links:
+
+- $\langle qo|op\rangle$: core package for GA-QAS. 
+   + [Github](https://github.com/vutuanhai237/qoop)
+   + [Wiki](https://github.com/vutuanhai237/qoop/wiki) 
+   + [Paper](https://doi.org/10.1016/j.softx.2024.101726)
+- GA-QAS: 
+   + Paper: Coming soon.
+   + [Wiki](https://github.com/vutuanhai237/qoop/wiki/GA%E2%80%90QAS:-folder-result): Load folder result and continue to evol()
+   + [Wiki](https://github.com/vutuanhai237/qoop/wiki/GA%E2%80%90QAS:-Full-pipeline): Full pipeline.
+
+Thanks for reading! Please do not hesitate to ask us any questions via e-mail: vutuanhai237@gmail.com or [LinkedIn](https://www.linkedin.com/in/vutuanhai237/).
+
+# References
+
+[1] Hai, V. T., Viet, N. T., & Ho, L. B. (2023). Variational preparation of entangled states on quantum computers. arXiv preprint arXiv:2306.17422.
+
+[2] Khatri, S., LaRose, R., Poremba, A., Cincio, L., Sornborger, A. T., & Coles, P. J. (2019). Quantum-assisted quantum compiling. Quantum, 3, 140.
+
+# Appendix: Full code
+
+The Jupyter notebook which contains the above code can be found [here](https://github.com/vutuanhai237/GA-QAS/blob/main/example/w_preparation.ipynb)
```

Comment:
I think it's best practice if you're going to link to code to use a permalink. E.g. https://github.com/vutuanhai237/GA-QAS/blob/694ff6a509df9acc7d3e8053ab89ccfd7d3127c0/example/w_preparation.ipynb. WDYT?

Reply:
Thanks for your suggestion, I changed it to a permalink.